zcash_client_backend/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this library adheres to Rust's notion of
   - `DormantMode`
 
 ### Changed
+- `zcash_client_backend::data_api`:
+  - `TargetValue`: An intent of representing spendable value to reach a certain 
+    targeted amount.
+  - `select_spendable_notes`: parameter `target_value` now is a `TargetValue`. 
+    Existing calls to this function that used `Zatoshis` now use 
+    `TargetValue::AtLeast(Zatoshis)`
 - Migrated to `arti-client 0.28`, `dynosaur 0.2`, `tonic 0.13`.
```

Comment:
Nit for the future @pacu: We generally put changelog entries for global things like dependency updates first, then go through the changes to modules in alphabetical order. So in this case, the spot to put your changelog entry would be between the two existing top-level bullet points.